.github/workflows/windows.yml

```diff
@@ -1,71 +1,67 @@
+      - name: Clone repository
+        uses: actions/checkout@master
 
-      - name: Setup python
-        uses: actions/setup-python@v1
+      - name: Install python
+        uses: actions/setup-python@master
         with:
           python-version: ${{ matrix.python-version }}
-          architecture: ${{ env.ARCH }}
 
-      - name: compile
+      - name: Compile planner
         shell: cmd
+        env:
+          CC: cl
+          CXX: cl
         run: |
-          "${{ env.VC }}" ${{ env.ARCH }} & python build.py ${{ matrix.build-version }}
-
-      - name: upload-planner
-        uses: actions/upload-artifact@master
-        with:
-          name: ${{ matrix.build-version }}
-          path: builds/${{ matrix.build-version }}
+          IF "${{ matrix.windows-version }}" == "windows-2016" (
+            "${{ env.VC2017 }}" ${{ env.ARCH }} & python build.py debug
+            "${{ env.VC2017 }}" ${{ env.ARCH }} & python build.py release
+          )
+          IF "${{ matrix.windows-version }}" == "windows-2019" (
+            "${{ env.VC2019 }}" ${{ env.ARCH }} & python build.py debug
+            "${{ env.VC2019 }}" ${{ env.ARCH }} & python build.py release
+          )
 
-
-  tests: # name of the job. Jobs run in parallel unless specified otherwise.
-    name: test
-    runs-on: windows-latest
-    needs: compile
-    strategy:
-      matrix:
-        python-version: ['3.6']
-    steps: # each - is a new sequentially run step
-      - uses: actions/checkout@master
-
-      - name: Setup python
-        uses: actions/setup-python@v1
-        with:
-          python-version: ${{ matrix.python-version }}
-          architecture: ${{ env.ARCH }}
-
-      - name: setup
+      - name: Install tox
         run: |
-          pip3 install pytest tox
-          mkdir builds
-
-      - name: download-planner
-        uses: actions/download-artifact@master
-        with:
-          # without 'name' attribute all artifacts are downloaded and the
-          # artifact name is used as directory name.
-          path: builds/
+          pip3 install tox
 
-      - name: test
+      - name: Run translator and search tests
         shell: cmd
+        env:
+          CC: cl
+          CXX: cl
+        # We do not run driver tests here because that would require
+        # VAL to be installed, which currently cannot be easily done
+        # on Windows for the version of VAL we use. When the maintainers
+        # of VAL fix the latest version to accept plans without time
+        # steps, we hope to be able to install VAL natively on Windows.
         run: |
           cd misc/
-          "${{ env.VC }}" ${{ env.ARCH }} & tox -e search,translator
+          IF "${{ matrix.windows-version }}" == "windows-2016" (
+            "${{ env.VC2017 }}" ${{ env.ARCH }} & tox -e translator,search
+          )
+          IF "${{ matrix.windows-version }}" == "windows-2019" (
+            "${{ env.VC2019 }}" ${{ env.ARCH }} & tox -e translator,search
+          )
```

Comment:
do your driver tests now also support Windows? If yes, can you please add them here to see if they work correctly.

Reply:
No, unfortunately not. We still need VAL. See comment above.